pages/api/offer.ts

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  try {
+    let offer = await (
+      await axios.get(`https://bootstrap.bolt12.org/fetchinvoice/lno1pg257enxv4ezqcneype82um50ynhxgrwdajx283qfwdpl28qqmc78ymlvhmxcsywdk5wrjnj36jryg488qwlrnzyjczs/100`)
+    ).data;
```

Comment:
```suggestion
    let offer = await (
      await axios.get(`https://bootstrap.bolt12.org/fetchinvoice/lno1pg257enxv4ezqcneype82um50ynhxgrwdajx283qfwdpl28qqmc78ymlvhmxcsywdk5wrjnj36jryg488qwlrnzyjczs/100`)
    ).data;
```
Wait a minite this is the pattern that we want avoid, in there you need to use the AppAPI instead